packs/feats/psychic-dedication.json

```diff
@@ -41,6 +41,13 @@
                 "path": "system.skills.occ.rank",
                 "value": 1
             },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.psychic.dedication.psiCantrips",
+                "priority": 10,
+                "value": []
+            },
```

Comment:
This would require a migration

Reply:
Sorry, I didn't realize this. I can't write it myself, unfortunately. Considering how busy you are, should I instead restore this to how it was in the first place in the meantime, or leave this be for now?

Reply:
I'll see if I can find time to write one, it's an easy one since it's a near carbon copy of an existing migration. Though I do wonder if there's a more clever way to do this...

Reply:
The original way is quite good except the Choice Sets are seemingly ignoring the predicates in their choices. Do you remember if that actually worked at the time? Maybe it's actually a bug and we don't need to touch this.

Reply:
I could swear they worked when I did them, but I really haven't had time to dig into it

Reply:
For the sake of my sanity, I went back and checked, and it looks like this part never worked (that is, surface cantrip appearing as a choice in the dedication, and not discarding an already taken standard cantrip from Psi Development)